src/Windows.Core/LIST_ENTRY.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+
+    [StructLayout(LayoutKind.Sequential)]
+    public unsafe struct LIST_ENTRY
+    {
+        public LIST_ENTRY* Flink;
```

Comment:
Scratch that It wouldn't work (at least by default) as `LOADED_IMAGE` contain an embeded instance of `LIST_ENTRY` and the marshaler would generate a pointer instead of embeding it if it was a class.

I wonder if there is a way to indicate that we don't want a pointer generated in this specific case.